client/my-sites/email/email-provider-features/button.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import Gridicon from 'calypso/components/gridicon';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+
+function EmailProviderFeaturesButton( { handleClick, isToggled } ) {
+	const translate = useTranslate();
+
+	return (
+		<Button className="email-provider-features__button" onClick={ handleClick }>
+			<span>{ translate( 'Learn more' ) }</span>
+
+			<Gridicon icon={ isToggled ? 'chevron-up' : 'chevron-down' } />
+		</Button>
+	);
+}
+
+EmailProviderFeaturesButton.propTypes = {
+	handleClick: PropTypes.func.isRequired,
+	isToggled: PropTypes.bool.isRequired,
```

Comment:
I originally refrained from using `isExpanded` or `isOpen` because I found them misleading: the appearance of the button changes when we click on it but it doesn't 'expand'. I had a look to existing components with a similar behavior: for `FormToggle` we use the term `checked` but I believe it's just to be consistent with how radio buttons and checkboxes work. I'll go with `isSwitched` but if you have better idea I'm all ears 🙂.